### PR TITLE
Add inset icon to country field; hide placeholder in modern theme; fix search layout in modern theme

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
This PR addresses 3 things:

1. Adds an inset icon to the country field (see design spec below).
2. Hides placeholder for country field when in modern theme.
3. Fixes the layout issue with search bar in modern theme.

image

### CODE REVIEW

File: src/app/public/modules/country-field/country-field.component.html [hunk 1]
```diff
@@ -68,3 +68,19 @@
...
   </span>
 </ng-template>
+
+<ng-template
+  #searchIconTemplateRef
+>
+  <div
+    class="sky-input-group-icon sky-input-box-icon-inset"
+  >
+    <sky-icon *skyThemeIf="'default'"
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/country-field/country-field.component.html:78]: Your code here has a `skyThemeIf` for `default` but the code in the component doesn't show this template at all in `deafult`. I don't think we need the `default` case here if we are never going to be showing it.

**Review by Blackbaud-TrevorBurch — CHANGES_REQUESTED** [review]
@Blackbaud-AlexKingman - one thing. Also, the build is failing so I haven't done any manual looking at the build yet.

File: src/app/public/modules/country-field/country-field.component.html (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/country-field/country-field.component.html:78]: Good point. This has been removed.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-country-field-search-button)

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**